blocks/record.go

```diff
@@ -0,0 +1,46 @@
+package blocks
+
+import (
+	"io"
+
+	"github.com/bsm/go-sparkey"
+)
+
+// A Record is one key/value pair loaded from a block.
+type Record struct {
+	ValueLen uint64
+
+	iterPool iterPool
+	iter     *sparkey.HashIter
+	io.Reader
```

Comment:
I thought I had a [test for it](https://github.com/stripe/sequins/blob/cc49a75f85918e567b73eafc152547b90e719d79/blocks/block_test.go#L57), but now that you say that I see how it's wrong.

I'm not actually sure the right way to do that. Should I add methods that proxy back to the iter?